driver/java/pom.xml

```diff
@@ -0,0 +1,87 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <!--The module should not rely on a parent-->
```

Comment:
We're planning to do a call on that next wednesday (will send an invite), but we agreed tentatively that we'd move it to the main code base, noting that we can move it back out if it is causing problems.

But I think the comment here is correct - the neo4j drivers are not versioned in lock-step with Neo4j itself - drivers need to be able to do patch releases and so on without a new Neo4j version, and drivers will support multiple versions of Neo4j. Drivers also do not need shared dependency management with Neo4j..